app/components/elements/tabs/pane_component.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= tag.div class: classes, id:, role: 'tabpanel', 'aria-labelledby': tab_id, tabindex: '0' do %>
```

Comment:
To cleanly support being able to dynamically choose the active pane, the tabbed interface had to be extracted into components. (The active pane was previously hardcoded, which didn't benefit from being componentized.)